content/en/real_user_monitoring/browser/data_collected.md

```diff
@@ -237,7 +237,7 @@ Detailed network timing data for the loading of an application's resources are c
 
 | Attribute       | Type   | Description                                                       |
 |-----------------|--------|-------------------------------------------------------------------|
-| `error.source`  | string | Where the error originates from (for example, `console` or `network`).     |
+| `error.source`  | string | Where the error originates from (for example, `console`).     |
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏What about adding a link to the paragraph with all the sources?